projects/go-lib/src/lib/components/go-datepicker/go-calendar.component.ts

```diff
@@ -137,9 +145,7 @@ export class GoCalendarComponent implements OnDestroy, OnInit, AfterViewChecked
     if (this.appendTo === 'body') {
       const calenderBodyPosition: any = this.calendarView.nativeElement.getBoundingClientRect();
       Object.assign(this.calendarView.nativeElement.style, {
-        height: `${calenderBodyPosition.height}px`,
-        bottom: `${calenderBodyPosition.bottom}px`,
-        top: `${calenderBodyPosition.top}px`,
```

Comment:
Can you please upload the Screen Shot once again please.

Reply:
@amitpatra91 sorry! It's uploaded now.

Reply:
I have updated the code because after click the arrow button of next and previous , This calendar auto Hight is not changing.

<img width="472" alt="date-picker-hegith-issue" src="https://user-images.githubusercontent.com/33975358/110101053-2229df00-7dc9-11eb-8920-a0ccb6a33bfe.PNG">

Reply:
I am getting proper alignment after adding this code. Please check that screen shot.

<img width="564" alt="datePicker-resolve" src="https://user-images.githubusercontent.com/33975358/110101916-273b5e00-7dca-11eb-8b7b-eef0e368036f.PNG">